### ISSUE TITLE
[PL][shaman][spell] Thunderstorm knockback sometimes doesnt work in pve

### ISSUE THREAD

**Issue by mrkariox** [Atlantiss/BugTracker#1748]
http://www.wowhead.com/spell=51490/thunderstorm

Spell nie dziala na odpychanie addow z Toxitrona (ODS) 
http://www.wowhead.com/npc=42897/poison-bomb

Nie dziala rowniez na przerywanie mind controla na Cho galu.
http://www.wowhead.com/spell=91303/conversion

**Comment by Jenovas** [comment]
Nie masz może przypadkiem Glyphu pod Thunderstorm?

**Comment by mrkariox** [comment]
Nie, mam jedynie ten pod redukcje cd o 10 sec. Na graczy czy inne mobki dziala normalnie.

**Comment by Wolffenstein** [comment]
Może te addy mają immune na knockback?

**Comment by Malylolek** [comment]
Nie mają bo taki typhon od druida na nie działa [ no chyba że to typhon jest zjebany ;d]

A co do ludzi oni też mają immune ? dziwne trochę nie przypominam sobie aby jakakolwiek klasa/rasa miała immune na knockback.

**Comment by mrkariox** [comment]
Nie maja immune na knockback, jak kolega wyzej wpomnial odepchniecie druida na nie dziala.

**Comment by qubuslabadz** [comment]
NA Cho'gallu jest immune na Thunderstorm, bugiem jest to. że Typhoon nie działa na to.

"Knockbacks and Deathgrip can now be used to interrupt players that are converted by Cho'gall."

Niestety, ale Thunderstorm != Typhoon.

**Comment by mrkariox** [comment]
Typhoon dziala na to, i to jest taki sam knockback jak thunder wiec albo wylaczamy typhoon, albo wlaczamy thunderstorm.

**Comment by qubuslabadz** [comment]
Przecież Ci napisałem, że Thunderstorm != Typhoon, co w tym niezrozumiałego? Osoba z conversion ma immune na knockbacki, ale Typhoon niestety omija to immune, bo działa inaczej niż Thunderstorm.

**Comment by Drutmaster** [comment]
Niestety od dwóch ostatnich ID Typhoon nie przerywa mind controla na Cho'gallu. Dmg wchodzi w ludzi, niestety nie ma odepchnięcia (nie mam glypha pod typhoona), przez co ludzie stoją dalej i ktoś inny musi im przerwać. Nie jest to może tragedia, nie mniej utrudnia to bardzo encounter w wersji hc. 
Rev. 4291

**Comment by qubuslabadz** [comment]
Jak nie przerywa, to dobrze działa.

**Comment by mrkariox** [comment]
W patchnotach z 4.1 jest mowa o tym że knockbacki powinny przerywac, a w dalszych patchnotach nie ma mowy o zadnym nerfie, wiec?
